how-tos/users-in-one-org.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: "Setting a Maximum Number of Users"
+
+---
+
+When creating a pricing structure for your product, it's common to limit a number of users in an organization depending 
+on the plan your customer is currently paying for. But how do you accomplish this in PropelAuth?
+
+## Setting a Max User Limit 
+
+Let's say you have a pricing model that has three tiers and plans
+
+- Hobbyist, meaning team size is max 1
+- Startup, meaning team size is max 5
+- Growth, meaning unlimited team size
+
+When a customer signs up for your product and selects a plan, you will want to assign the team to their own organization with 
```

Comment:
This feels a little confusing, maybe just because "team" is in there? I think you can just be more direct and say "When an organization selects a plan, you will want to set a limit based on the plan they are paying for"